### ISSUE TITLE
Enhancement request: Extra invalid syntax detection

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#984]
I've come across following in our code.
This is wrong because of the double `&&` on `&not_avail_string`

```
{ cceinc/trigdsp.i &ident      = BOEKJAAR-MISC
            &rec_field  = BJ-EINDE
            &dsp_frame  = {&misc_frame}
            &dsp_field  = WZ-TOT-DAT
            &&not_avail_string=ltodaystring }
```

This is easily overlooked and goes undetected because in this case it is an optional parameter.

Is the double & something that could be detected by a rule ?

I cannot imagine a situation where this would be valid, but maybe my imagination is limited.

**Comment by gquerret** [comment]
It can be detected, but that will trigger false positives on some codebase (I'm sure there is at least one app on Earth that considers that having named parameters starting with `&` is a good idea). Could be part of a new optional rule.

**Comment by gquerret** [comment]
> Is the double & something that could be detected by a rule ?

The InvalidNamedParameter rule does this kind of check on named include parameters.
